tests/Issue2284.py

```diff
@@ -0,0 +1,87 @@
+from unittest import mock
+
+from . import Framework
+
+
+class Issue2284(Framework.TestCase):
```

Comment:
Please add the tests to the respective class tests like `tests/Repository` or `tests/Repository`.

Reply:
Hi @EnricoMi, I have followed the pattern that I can already see in the repo.

I've had a look and to add my tests into the test/repository file would require some refactoring to be able to regenerate the replay files against my org/repos. 

What do you advice I do?